recipes/cairomm/all/conanfile.py

```diff
@@ -25,170 +35,118 @@ class CairommConan(ConanFile):
         "fPIC": True,
     }
 
-    generators = "pkg_config"
-    exports_sources = "patches/**"
-    short_paths = True
-
-    def _abi_version(self):
-        return "1.16" if tools.Version(self.version) >= "1.16.0" else "1.0"
-
-    def validate(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self):
-            raise ConanInvalidConfiguration("Cross-building not implemented")
-        if self.settings.compiler.get_safe("cppstd"):
-            if self._abi_version() == "1.16":
-                tools.check_min_cppstd(self, 17)
-            else:
-                tools.check_min_cppstd(self, 11)
-        if self.options.shared and not self.options["cairo"].shared:
-            raise ConanInvalidConfiguration(
-                "Linking against static cairo would cause shared cairomm to link "
-                "against static glib which can cause problems."
-            )
-
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _abi_version(self):
+        return "1.16" if Version(self.version) >= "1.16.0" else "1.0"
```

Comment:
This does not introduce too much variation complexity, so I'm ok with keeping the old version around for a bit longer, thanks for doing the extra work to keep its support when it would have been easier to just don't support 1.14 :)